specification/ml/_types/TrainedModel.ts

```diff
@@ -131,34 +144,49 @@ export class TrainedModelSizeStats {
 
 export class TrainedModelDeploymentNodesStats {
   /** The average time for each inference call to complete on this node. */
-  average_inference_time_ms: DurationValue<UnitFloatMillis>
+  average_inference_time_ms?: DurationValue<UnitFloatMillis>
+
+  average_inference_time_ms_last_minute?: DurationValue<UnitFloatMillis>
+
+  /** The average time for each inference call to complete on this node, excluding cache */
+  average_inference_time_ms_excluding_cache_hits?: DurationValue<UnitFloatMillis>
+
   /** The number of errors when evaluating the trained model. */
-  error_count: integer
+  error_count?: integer
   /** The total number of inference calls made against this node for this model. */
-  inference_count: integer
+  inference_count?: long
+
+  inference_cache_hit_count?: long
+
+  inference_cache_hit_count_last_minute?: long
+
   /** The epoch time stamp of the last inference call for the model on this node. */
-  last_access: long
+  last_access?: EpochTime<UnitNanos>
   /**
    * Information pertaining to the node.
    * @availability stack
    */
-  node: DiscoveryNode
+  node?: DiscoveryNode
   /**
    * The number of allocations assigned to this node.
    */
-  number_of_allocations: integer
+  number_of_allocations?: integer
   /** The number of inference requests queued to be processed. */
-  number_of_pending_requests: integer
+  number_of_pending_requests?: integer
+
+  peak_throughput_per_minute: long
   /** The number of inference requests that were not processed because the queue was full. */
-  rejection_execution_count: integer
+  rejection_execution_count?: integer
   /** The current routing state and reason for the current routing state for this allocation. */
   routing_state: TrainedModelAssignmentRoutingTable
   /** The epoch timestamp when the allocation started. */
-  start_time: EpochTime<UnitMillis>
+  start_time?: EpochTime<UnitNanos>
```

Comment:
Per https://github.com/elastic/elasticsearch/blob/045f6a31f994f51d87a217be60251e060132c8a1/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/assignment/AssignmentStats.java#L314-L316:

```suggestion
  start_time?: EpochTime<UnitMillis>
```